packages/xo-web/src/xo-app/dashboard/health/index.js

```diff
@@ -620,6 +628,15 @@ export default class Health extends Component {
     )
   )
 
+  _getPoolsWithNoDefaultSr = createCollectionWrapper(
+    createSelector(
+      () => this.props.pools,
+      () => this._getPoolIds(),
+      (pools, poolIds) =>
+        filter(isEmpty(poolIds) ? pools : pick(pools, poolIds), ({ default_SR }) => default_SR === undefined)
```

Comment:
Shall we create a selector for `isEmpty(poolIds) ? pools : pick(pools, poolIds)`?